modules/apps/commerce/commerce-payment-service/src/main/java/com/liferay/commerce/payment/internal/entry/CommercePaymentEntryRefundTypeRegistryImpl.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.commerce.payment.internal.entry;
+
+import com.liferay.commerce.payment.entry.CommercePaymentEntryRefundType;
+import com.liferay.commerce.payment.entry.CommercePaymentEntryRefundTypeRegistry;
+import com.liferay.commerce.payment.internal.entry.comparator.CommercePaymentEntryRefundTypeOrderComparator;
+import com.liferay.osgi.service.tracker.collections.list.ServiceTrackerList;
+import com.liferay.osgi.service.tracker.collections.list.ServiceTrackerListFactory;
+import com.liferay.portal.kernel.feature.flag.FeatureFlagManagerUtil;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.util.Validator;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+import org.osgi.framework.BundleContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+
+/**
+ * @author Alessio Antonio Rendina
+ */
+@Component(service = CommercePaymentEntryRefundTypeRegistry.class)
+public class CommercePaymentEntryRefundTypeRegistryImpl
+	implements CommercePaymentEntryRefundTypeRegistry {
+
+	@Override
+	public CommercePaymentEntryRefundType getCommercePaymentEntryRefundType(
+		String key) {
```

Comment:
Since here we want the specific one (we have the key param), registering a ServiceTrackerMap<String, Object> instead of a ServiceTrackerList was simplier.

Reply:
Ok, fixed it. Thank you!